src/components/tabset/tabset.component.spec.ts

```diff
@@ -116,12 +117,35 @@ describe('Tabset Component', () => {
         expect(component.onTabDeselect).toHaveBeenCalled();
     });
 
+    it('should change to new tab when tab is programmatically selected', () => {
+
+        const tab1 = getTabForClass(0, nativeElement);
+        const tab2 = getTabForClass(2, nativeElement);
```

Comment:
To avoid confusion, perhaps this should be named `tab3` since you're selecting the 3rd tab?